samples/MvcSample.Web/Controllers/JsonPatchController.cs

```diff
@@ -34,8 +35,13 @@ public IActionResult Patch([FromBody] JsonPatchDocument<Customer> patchDoc)
                 }
             };
 
-            patchDoc.ApplyTo(customer);
+            patchDoc.ApplyTo(customer, ModelState);
 
+            if (!ModelState.IsValid)
+            {
+                var entry = ModelState.SingleOrDefault(m => m.Key.Contains("JsonPatchDocument"));
+                return new ObjectResult(entry.Value.Errors[0].ErrorMessage);
```

Comment:
We'd want this to work with `return BadRequest(ModelState)` without any massaging.